makefiles/docker.inc.mk

```diff
@@ -5,13 +5,11 @@
 # When the docker image is updated, checks at
 # dist/tools/buildsystem_sanity_check/check.sh start complaining in CI, and
 # provide the latest values to verify and fill in.
-DOCKER_TESTED_IMAGE_ID := 488d97b1b8285ca6a9518f177126fc5ab077135648ef40c1ac7867d16ac2d640
 DOCKER_TESTED_IMAGE_REPO_DIGEST := beb980f9ef00e7143fba913f3cb927866f987fe570141e145539c42141a940bd
 
 DOCKER_PULL_IDENTIFIER := docker.io/riot/riotbuild@sha256:$(DOCKER_TESTED_IMAGE_REPO_DIGEST)
-DOCKER_IMAGE_DEFAULT := sha256:$(DOCKER_TESTED_IMAGE_ID)
 DOCKER_AUTO_PULL ?= 1
```

Comment:
```suggestion
```

Since it's not used anymore. The default way to skip the docker pull (Brandenburg train type of scenario) is defining `DOCKER_IMAGE=docker.io/riot/riotbuild:latest` (and use an outdated/untested docker image), right?

Reply:
Done and squashed